skimage/measure/fit.py

```diff
@@ -573,6 +573,10 @@ def ransac(data, model_class, min_samples, residual_threshold,
         where the probability (confidence) is typically set to a high value
         such as 0.99, and e is the current fraction of inliers w.r.t. the
         total number of samples.
+    exceptions : exception class or tuple of exception classes
```

Comment:
Hm, this would not work. Since RANSAC is randomly sampling and the chance of hitting a degenerate case is pretty high. So there is not really anything the user can do about that... One way would be to return NAN matrices in `_center_and_normalize_points` and silently failing this way.